lib/utils/util.js

```diff
@@ -184,72 +183,13 @@ util.encodeU32 = function encodeU32(num) {
   return buf;
 };
 
-/**
- * Create a valid mtx with limited
- * number of outputs
- * @param {Number} limit
- * @param {Map<String, Array<T>>} domainOutputMap
- * @returns {{
- *   validDomains: Array<{String, Number}>,
- *   invalidDomains: Array<{String, Number}>
- * }}
- */
-
-util.createBatch = function(limit, domainOutputMap) {
-  if (!limit || limit === 0)
-    throw new Error('invalid limit provided!');
-
-  if (!domainOutputMap || domainOutputMap.entries().length === 0) {
-      throw new Error('Invalid map provided!');
-  }
-
-  const entriesArray = [...domainOutputMap.entries()];
-  entriesArray.sort((elem1, elem2) => elem1[1].length > elem2[1].length);
-
-  const rejectedDomains = [];
-  const validDomains = [];
-  let totalNumberOfOutputs = 0;
-
-  for (const [name, outputs] of entriesArray) {
-      const outputLen = totalNumberOfOutputs + outputs.length;
-      if (outputLen > limit) {
-        // if there is any space left, partially process the bids
-        // of this domain
-        const availableSpots = limit - totalNumberOfOutputs;
-        if (availableSpots > 0) {
-          totalNumberOfOutputs += availableSpots;
-          validDomains.push({name, bidCount: availableSpots});
-          const remainingBidsForDomain = outputs.length - availableSpots;
-          rejectedDomains.push({name, bidCount: remainingBidsForDomain});
-        } else {
-          rejectedDomains.push({name, bidCount: outputs.length});
-        }
-      } else {
-          totalNumberOfOutputs += outputs.length;
-          validDomains.push({name, bidCount: outputs.length});
-      }
-  }
-
-  return {validDomains: validDomains, rejectedDomains: rejectedDomains};
-};
-
-/**
- * Create a valid mtx with strictly limited
- * number of outputs, not partial reveals permitted
- * @param {Number} limit
- * @param {Map<String, Array<T>>} domainOutputMap
- * @returns {{
- *   validDomains: Array<{String, Number}>,
- *   invalidDomains: Array<{String, Number}>
- * }}
- */
-
-util.createStrictBatch = function(limit, domainOutputMap) {
-  if (!limit || limit === 0)
-    throw new Error('invalid limit provided!');
+util.createStrictBatch = function (limit, domainOutputMap) {
```

Comment:
Is this still needed? From my grepping seems like this is just for batch reveals which are in the plugin now right?

Reply:
that is definitely not needed no one is using it

Reply:
correction it is needed in the wallet.js file, i have moved it back (for batch reveals)